Assembler.Base/RawFrameMessageEnricher.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Assembler.Core;
+using Assembler.Core.Entities;
+using Assembler.Core.RawAssemblingEntities;
+
+namespace Assembler.Base
+{
+    public class RawFrameMessageEnricher : IMessageEnricher<BaseFrame, RawMessageInAssembly>
+    {
+        public void Enrich(BaseFrame frame, RawMessageInAssembly message)
+        {
+            message?.AssembledFrames.Add(frame);
```

Comment:
In which case does the message can come null? I think that if it happens - for some reason - you dont need to let it pass like that. Remove the question mark and let an exception to be thrown

Reply:
I used casting before and forgot to remove it, you are right